build.rs

```diff
@@ -64,8 +68,22 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .map_err(|_| "Unable to generate bindings")?
         .write_to_file(out_dir.join("bindings.rs"))?;
 
-    // Generate the protobuf definition
-    prost_build::compile_protos(&[&out_protobuf_path.join(LIBRARY_NAME).with_extension("proto")], &[&out_protobuf_path])?;
+    let protoc_exists = Command::new("protoc").arg("--version").status().is_ok();
+    if env::var("REGENERATE_PROTOBUF").is_ok() || protoc_exists {
+        println!("generating protobuf bindings");
+        // HACK: Set OUT_DIR to src/ so that the generated protobuf file is copied to src/protobuf.rs
+        let src_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR")?).join("src");
+        env::set_var("OUT_DIR", &src_dir);
+
+        prost_build::compile_protos(&[&out_protobuf_path.join(LIBRARY_NAME).with_extension("proto")], &[&out_protobuf_path])?;
+
+        std::fs::rename(src_dir.join("pg_query.rs"), src_dir.join("protobuf.rs"))?;
```

Comment:
It seems like this line broke the docs.rs build https://docs.rs/crate/pg_query/6.1.0/builds/1945029

Reply:
Maybe we should detect the `DOCS_RS` envvar like https://github.com/polywrap/rust-wrap-client/pull/245 does

Reply:
pr with the fix is up: https://github.com/pganalyze/pg_query.rs/pull/56